src/App.js

```diff
@@ -75,14 +75,30 @@ class App extends Component {
     this.props.setItemToStorage("lastSelectedLanguage", language);
   };
 
+  getBrowserLanguageOrNull = () => {
+    const browserLanguage = navigator.language.slice(0, 2).toUpperCase();
+    const isBrowserLangSupported = Object.keys(AVAILABLE_LANGUAGES).some(
+      lang => lang === browserLanguage
+    );
```

Comment:
this should also work, right ?

```suggestion
    const isBrowserLangSupported = AVAILABLE_LANGUAGES[browserLanguage];
```